server/src/main/java/org/apache/uniffle/server/ShuffleTaskManager.java

```diff
@@ -308,6 +310,21 @@ protected static int getMaxConcurrencyWriting(
   public StatusCode cacheShuffleData(
       String appId, int shuffleId, boolean isPreAllocated, ShufflePartitionedData spd) {
     refreshAppId(appId);
+    long partitionSize = getPartitionDataSize(appId, shuffleId, spd.getPartitionId());
+    long deltaSize = 0L;
+    for (ShufflePartitionedBlock spb : spd.getBlockList()) {
```

Comment:
You can use `spd.getTotalBlockSize()`

Reply:
Great! Thanks for remind me this method.

And I found that is pure getter but a sum calculator, I put this calculate into constructor of `ShufflePartitionedData` now